Rakefile

```diff
@@ -70,7 +70,7 @@ def install_pods_in_example_project()
 end
 
 def get_changed_files()
-  `git status --porcelain | grep -Ev "CHANGELOG|UNRELEASED"`.lines
+  `git status --porcelain | grep -Ev "CHANGELOG|UNRELEASED|package-lock.json"`.lines
```

Comment:
I don't know why `package-lock` wasn't already excluded - we exclude it in the other repos!